dcpy/test/lifecycle/package/resources/colp_single_feature_package/metadata.yml

```diff
@@ -299,3 +311,11 @@ destinations:
     overrides:
       display_name: "display_name overridden at the destination level"
       destination_file_name: shapefile_blob_{{ version }}
+  # BYTES
+  - id: bytes
+    type: bytes
```

Comment:
nit since this really isn't being used. But is BYTES a "bytes" type destination? I would think of it more as "manual" given that the update process is currently out of our hands and involves sending data manually to web team (or maybe it's a `web_team` type then).

Reply:
I do think Bytes is a destination, even if we don't push to it ourselves. I've been conceiving of it as the noun describing where the dataset lives, either where we'd push to or pull from. (Right now we're just using it pull, but honestly I don't see why, in the future, we wouldn't just use it to automate the push as well.)

Framing it another way, I'm envisioning a future where `destination.type` is just used to to dispatch to a given connector, to either push or pull, and the heuristic I'm using for what to call the `destination.type` is: "what would we call the connector." So in this case, Bytes.